obspy/clients/fdsn/__init__.py

```diff
@@ -34,38 +34,42 @@
 >>> from obspy.clients.fdsn.header import URL_MAPPINGS
 >>> for key in sorted(URL_MAPPINGS.keys()):
 ...     print("{0:<11} {1}".format(key,  URL_MAPPINGS[key]))  # doctest: +SKIP
-AUSPASS     http://auspass.edu.au
-BGR         http://eida.bgr.de
-EARTHSCOPE  http://service.iris.edu
+AUSPASS     https://auspass.edu.au
+BGR         https://eida.bgr.de
+BGS         https://eida.bgs.ac.uk
+EARTHSCOPE  https://service.iris.edu
 EIDA        http://eida-federator.ethz.ch
-EMSC        http://www.seismicportal.eu
-ETH         http://eida.ethz.ch
-GEOFON      http://geofon.gfz-potsdam.de
-GEONET      http://service.geonet.org.nz
+EMSC        https://www.seismicportal.eu
+ETH         https://eida.ethz.ch
+GEOFON      https://geofon.gfz-potsdam.de
+GEONET      https://service.geonet.org.nz
 GFZ         http://geofon.gfz-potsdam.de
-ICGC        http://ws.icgc.cat
+ICGC        https://ws.icgc.cat
 IESDMC      http://batsws.earth.sinica.edu.tw
-INGV        http://webservices.ingv.it
+IGN         http://fdsnws.sismologia.ign.es
+INGV        https://webservices.ingv.it
 IPGP        http://ws.ipgp.fr
-IRIS        http://service.iris.edu
-IRISPH5     http://service.iris.edu
-ISC         http://www.isc.ac.uk
+IRIS        https://service.iris.edu
+IRISDMC     https://service.iris.edu
+IRISPH5     https://service.iris.edu
+ISC         https://www.isc.ac.uk
+KAGSR       http://sdis.emsd.ru
 KNMI        http://rdsa.knmi.nl
-KOERI       http://eida.koeri.boun.edu.tr
+KOERI       https://eida.koeri.boun.edu.tr
 LMU         https://erde.geophysik.uni-muenchen.de
-NCEDC       http://service.ncedc.org
-NIEP        http://eida-sc3.infp.ro
-NOA         http://eida.gein.noa.gr
+NCEDC       https://service.ncedc.org
+NIEP        https://eida-sc3.infp.ro
+NOA         https://eida.gein.noa.gr
 NRCAN       https://earthquakescanada.nrcan.gc.ca
-ODC         http://www.orfeus-eu.org
-ORFEUS      http://www.orfeus-eu.org
+ODC         https://www.orfeus-eu.org
+ORFEUS      https://www.orfeus-eu.org
 RASPISHAKE  https://data.raspberryshake.org
-RESIF       http://ws.resif.fr
+RESIF       https://ws.resif.fr
 RESIFPH5    http://ph5ws.resif.fr
-SCEDC       http://service.scedc.caltech.edu
+SCEDC       https://service.scedc.caltech.edu
 TEXNET      http://rtserve.beg.utexas.edu
-UIB-NORSAR  http://eida.geo.uib.no
-USGS        http://earthquake.usgs.gov
+UIB-NORSAR  https://eida.geo.uib.no
+USGS        https://earthquake.usgs.gov
 USP         http://sismo.iag.usp.br
```

Comment:
Some of the ones that are left as "http" seem to also support "https".. like e.g. https://sismo.iag.usp.br/fdsnws/dataselect/1

Note that not all implementations of FDSNWS bother to show nice info pages on any or all sub-routes, like sometimes just doing "http.../fdsnws/" will show an empty page